lib/rubocop/cop/rspec/hooks_before_examples.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks for before/around/after hooks that come after an example.
+      #
+      # @example
+      #   # Bad
+      #
+      #   it 'checks what foo does' do
+      #     expect(foo).to be
+      #   end
+      #
+      #   before { prepare }
+      #   after { clean_up }
+      #
+      #   # Good
+      #   before { prepare }
+      #   after { clean_up }
+      #
+      #   it 'checks what foo does' do
+      #     expect(foo).to be
+      #   end
+      #
+      class HooksBeforeExamples < Cop
+        include RangeHelp
+        include RuboCop::RSpec::FinalEndLocation
+
+        MSG = 'Move `%<hook>s` before the examples in the group.'.freeze
```

Comment:
I’m not sure if “before” is the right word here. Above? Atop? @dgollahon can you help us?

Reply:
I borrowed the wording from `LetBeforeExamples` but for the `before` hook it sounds weird indeed. LedingSubject use `above` so I'll use it